snet_sdk/scripts/blockchain

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Comment:
This is the same file as snet_cli/scripts/blockchain. Why not just reuse it

Reply:
Done